schemas/bridging/broadcastAgentRequest.schema.json

```diff
@@ -45,9 +45,7 @@
           "properties": {
             "requestUuid": true,
             "timestamp": true,
-            "source": {
```

Comment:
I think broadcasts alawys come from apps... need the metadata for originatingAppMetadata, hence, appID should be required in the output typescript